RevenueCatUI/Templates/V2/Components/Icon/IconComponentViewModel.swift

```diff
@@ -122,10 +167,6 @@ struct IconComponentStyle {
     let iconBackgroundBorder: ShapeModifier.BorderInfo?
     let iconBackgroundShadow: ShadowModifier.ShadowInfo?
 
-//    shape: PaywallComponent.Shape?,
-//    border: PaywallComponent.Border?,
-//    shadow: PaywallComponent.Shadow?,
```

Comment:
… these aren't used?

Reply:
Ahhh, these I think were left over copy/paste from when this was first created 😅 There are other border and shadow properties that are being used on icon